src/store/multi.rs

```diff
@@ -8,6 +8,8 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
+#![allow(clippy::trivially_copy_pass_by_ref)]
```

Comment:
Perhaps we should fix these instead of suppressing the warning. Any reason not to do so?

Reply:
Good question :)

I recall that I've fixed them in https://github.com/mozilla/rkv/pull/88, but those stubborn references came back again perhaps with the recent commits.

Let's fix them again.